### PR TITLE
Included documentation for IRMA

### PR DESCRIPTION
**Pull request by Shettland**
## PR checklist

- [X] This PR aims to provide documentation markdowns for IRMA to be used in bioinfo-doc , including output and workflow. Related to #4 and BU-ISCIII's issue [#46](https://github.com/BU-ISCIII/BU-ISCIII/issues/46)
- [ ] Make sure your code lints (`black and flake8`).
- If a new tamplate was added make sure:
    - [X] Template's schema is added in `templates/services.json`.
    - [X] Template's pipeline's documentation in `assets/reports/md/template.md` is added.
    - [X] Results Documentation in `assets/reports/results/template.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).
- [ ] If you know a new user was added to the SFTP, make sure you added it to `templates/sftp_user.json`

### CODE REVIEW

File: bu_isciii/assets/reports/md/IRMA.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+![](images/mqc_fastp_plot.png)
+
+## IRMA workflow
+
+[IRMA (Iterative Refinement Meta-Assembler)](https://wonder.cdc.gov/amd/flu/irma/) was designed for the robust assembly, variant calling, and phasing of highly variable RNA viruses. IRMA is deployed for this service to analyse Influenza virus. IRMA is free to use and parallelizes computations for both cluster computing and single computer multi-core setups. You can read the [IRMA manuscript](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3030-6#Sec9) for more background on the methodology. 
+
+![](images/IRMA_workflow.png)
+
+**IRMA** uses several auxiliar softwares in its workflow:
+- [BLAT](http://www.kentinformatics.com/products.html) for the match step
```
Comment (svarona) [inline bu_isciii/assets/reports/md/IRMA.md:49]: Lists must be separated by new line

File: bu_isciii/assets/reports/md/IRMA.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+**Final files:**
+>- **_gene\_segment_.bam**			Sorted BAM file for the final gene_segment assembly (merged if applicable)
+>- **_gene\_segment_.bam.bai**			BAM file index for gene_segment assembly
+>- **_gene\_segment_.fasta**			Final assembled plurality consensus (no mixed base calls) for gene_segment
+>- **_gene\_segment_.a2m**			Optional file: Plurality consensus aligned to profile HMM
+>- **_gene\_segment_.vcf**			Custom variant call file for called IRMA variants for each gene segment
+
+**Folders:**
+><u>**amended_consensus/</u>**: Assembled consensus per gene segment with missed based calls. Numbers correspond to fragments in $SEG_NUMBERS
```
Comment (svarona) [inline bu_isciii/assets/reports/md/IRMA.md:70]: This is looking awful in the GitHub preview....

File: bu_isciii/assets/reports/md/IRMA.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+# IRMA
+
+IRMA does not include quality trimming of the reads by itself. Therefore, before the execution of the pipeline the samples are analysed with FastQC for quality control and pre-processed with fastp.
```
Comment (saramonzon) [inline bu_isciii/assets/reports/md/IRMA.md:3]: Here you need to describe the software with the link to the source etc..

Comment (saramonzon) [reply bu_isciii/assets/reports/md/IRMA.md:3]: this info can be in the next section

Comment (saramonzon) [reply bu_isciii/assets/reports/md/IRMA.md:3]: i see its in the irma section, here you can just add a TOC and just that :)

File: bu_isciii/templates/services.json [hunk 4]
```diff
@@ -319,8 +319,8 @@
       },
       "no_copy": ["RAW", "TMP"],
       "last_folder":"REFERENCES",
-      "delivery_md": "",
-      "results_md": ""
+      "delivery_md": "IRMA.md",
```
Comment (saramonzon) [inline bu_isciii/templates/services.json:322]: use low case for irma in the name of the file please

**Review by saramonzon — APPROVED** [review]

**Pull request merged**